packages/compass-import-export/src/modules/export.spec.js

```
@@ -1,68 +1,69 @@
-import configureExportStore from '../stores/export-store';
-import { ConnectionOptions, DataService } from 'mongodb-data-service';
+import store from '../stores/export-store';
+import { DataService } from 'mongodb-data-service';
 import { promisify } from 'util';
 import { once } from 'events';
+import { dataServiceConnected, globalAppRegistryActivated } from './compass';
 describe('export [module]', () => {
+  beforeEach(() => {
+    store.dispatch(actions.reset());
+  });
+
   describe('#reducer', () => {
     context('#startExport', () => {
-      let store;
-      const localAppRegistry = new AppRegistry();
       const globalAppRegistry = new AppRegistry();
       const dataService = new DataService({ connectionString: 'mongodb://localhost:27018/local'});
       const createCollection = promisify(dataService.createCollection).bind(dataService);
       const dropCollection = promisify(dataService.dropCollection).bind(dataService);
       const insertMany = promisify(dataService.insertMany).bind(dataService);
       const TEST_COLLECTION_NAME = 'local.foobar';
-      
+
       afterEach(async function() {
         await dropCollection(TEST_COLLECTION_NAME);
       });
 
       beforeEach(async function() {
         await dataService.connect();
-        try {
-          await createCollection(TEST_COLLECTION_NAME);
-          await insertMany(TEST_COLLECTION_NAME, [
-            {
-              _id: 'foo',
-              first_name: 'John',
-              last_name: 'Appleseed'
-            }
-          ]);
-        } catch (err) {
-          console.log(err);
-        }
 
-        store = configureExportStore({
-          localAppRegistry: localAppRegistry,
-          globalAppRegistry: globalAppRegistry,
-          namespace: TEST_COLLECTION_NAME,
-          dataProvider: {
-            error: function(err) { throw err; },
-            dataProvider: dataService
+        await createCollection(TEST_COLLECTION_NAME);
```

Nice, I like we're not silently ignoring the errors now.